ppl/src/main/antlr/OpenSearchPPLParser.g4

```diff
@@ -363,15 +363,7 @@ wcFieldList
    ;
 
 sortField
-   : (PLUS | MINUS)? sortFieldExpression
-   ;
-
-sortFieldExpression
```

Comment:
@currantw was this `sortFieldExpression` used in a query anywhere ?

Reply:
it is used in `sortField`, which is used by `Trendline` and `SORT BY`.  @currantw we sure we want to include these changes?

Reply:
As discussed, I've reverted these changes in favour of addressing this independently in #3180.